charts/jaeger/templates/allinone-sa.yaml

```diff
@@ -1,14 +0,0 @@
-{{- if .Values.allInOne.enabled -}}
```

Comment:
why is this deleted? all-in-one is still a valid deployment and will need a ServiceAccount